osv-scanner.toml

```diff
@@ -0,0 +1,122 @@
+ecosystem = "Go"
+# Override the license to an allowed one until https://github.com/google/osv-scanner/issues/1124 is resolved and we can skip it from licnese scanning instead
+license.override = ["Apache-2.0"]
+reason = "This package has MPL-2.0 which is not approved in CNCF Allowlist, but it has an exception. See https://github.com/cncf/foundation/blob/main/license-exceptions/CNCF-licensing-exceptions.csv"
+
+[[PackageOverrides]]
+name = "github.com/hashicorp/go-version"
+version = "1.7.0"
+ecosystem = "Go"
+# Override the license to an allowed one until https://github.com/google/osv-scanner/issues/1124 is resolved and we can skip it from licnese scanning instead
+license.override = ["Apache-2.0"]
+reason = "This package has MPL-2.0 which is not approved in CNCF Allowlist, but it has an exception. See https://github.com/cncf/foundation/blob/main/license-exceptions/CNCF-licensing-exceptions.csv"
+
+[[PackageOverrides]]
+name = "github.com/hashicorp/hcl"
+version = "1.0.0"
+ecosystem = "Go"
+# Override the license to an allowed one until https://github.com/google/osv-scanner/issues/1124 is resolved and we can skip it from licnese scanning instead
+license.override = ["Apache-2.0"]
+reason = "This package has MPL-2.0 which is not approved in CNCF Allowlist, but it has an exception. See https://github.com/cncf/foundation/blob/main/license-exceptions/CNCF-licensing-exceptions.csv"
+
+[[PackageOverrides]]
+name = "github.com/moby/patternmatcher"
+version = "0.6.0"
+ecosystem = "Go"
+license.override = ["Apache-2.0"]
+reason = "Unidentified license, remove once https://github.com/google/deps.dev/issues/106 is resolved"
+
+[[PackageOverrides]]
+name = "github.com/opencontainers/go-digest"
+version = "1.0.0"
+ecosystem = "Go"
+license.override = ["Apache-2.0"]
+reason = "This package has dual license - the code is licensed under the Apache 2.0 license and the docs under CC-BY-SA-4.0 license"
+
+[[PackageOverrides]]
+name = "github.com/shoenig/go-m1cpu"
+version = "0.1.6"
+ecosystem = "Go"
+# Override the license to an allowed one until https://github.com/google/osv-scanner/issues/1124 is resolved and we can skip it from licnese scanning instead
+license.override = ["Apache-2.0"]
+reason = "This package has MPL-2.0 which is not approved in CNCF Allowlist, but it has an exception. See https://github.com/cncf/foundation/blob/main/license-exceptions/cncf-exceptions-2023-08-31.spdx"
+
+[[PackageOverrides]]
+name = "stdlib"
+ecosystem = "Go"
+license.override = ["BSD-3-Clause"]
+reason = "Unidentified license, remove once https://github.com/google/deps.dev/issues/86 is resolved"
+
+[[PackageOverrides]]
+name = "github.com/grafana/tempo"
+version = "1.5.0"
+ecosystem = "Go"
+# Override the license to an allowed one until https://github.com/google/osv-scanner/issues/1124 is resolved and we can skip it from licnese scanning instead
+license.override = ["Apache-2.0"]
+reason = "This package is only used in e2e tests so we can ignore its license"
```

Comment:
we change change it to zipkin or other backend.
does loki has same problem?

Reply:
Loki has the same `AGPL-3.0` license.